NEWS.md

```diff
@@ -6,6 +6,8 @@
 
 * `NA` levels in factors aren't dropped when passed to `recipe()`. (#1291)
 
+* `recipe()` no longer crashes when given long formula expression. (#1283)
```

Comment:
Looks like you generally don't prefer this elsewhere in NEWS, but noting tidy style is like so:

```suggestion
* `recipe()` no longer crashes when given long formula expression (#1283).
```